sass/_Quickview.scss

```diff
@@ -9,9 +9,31 @@
     @include retina-sprite($quickview-group);
   }
 
-  .coveo-caption-for-quickview {
+  &:hover .coveo-caption-for-icon {
+    display: inline;
+  }
+  
+  .coveo-caption-for-icon {
     font-size: $font-size-smallest;
-    margin-top: -7px;
+    display: none;
+    background: $color-greyish-dark-blue;
+    color: white;
+    border-radius: 2px;
+    padding: 6px 16px;
+    position: absolute;
+    white-space: nowrap;
+    transform: translate(calc(-50% - 11px), calc(100% - 6px));
```

Comment:
Have you been able to test it with Safari/IE 11 ?

I think I remember those browsers having issue displaying `calc` css rules properly, but I'm not 100% sure.